vm/src/stdlib/imp.rs

```diff
@@ -25,6 +30,53 @@ fn imp_is_builtin(name: PyStringRef, vm: &VirtualMachine) -> bool {
     vm.stdlib_inits.borrow().contains_key(name.as_str())
 }
 
+fn imp_is_frozen(name: PyStringRef, vm: &VirtualMachine) -> bool {
+    vm.frozen.borrow().contains_key(name.as_str())
+}
+
+fn imp_create_builtin(spec: PyObjectRef, vm: &VirtualMachine) -> PyResult {
+    let sys_modules = vm.get_attribute(vm.sys_module.clone(), "modules").unwrap();
+
+    let name = &objstr::get_value(&vm.get_attribute(spec.clone(), "name")?);
+
+    if let Ok(module) = sys_modules.get_item(name, vm) {
+        Ok(module)
+    } else {
+        if let Some(make_module_func) = vm.stdlib_inits.borrow().get(name) {
+            Ok(make_module_func(vm))
+        } else {
+            Ok(vm.get_none())
+        }
+    }
+}
+
+fn imp_exec_builtin(_mod: PyModuleRef, _vm: &VirtualMachine) -> i32 {
+    // TOOD: Should we do something here?
```

Comment:
It might be nice to use `unimplemented` here, or to log an error or warning with a logger?

Reply:
I am actually not sure if we should do something there... The CPython function `PyModule_ExecDef` does some sort of initializing that I don't think we currently need to do.